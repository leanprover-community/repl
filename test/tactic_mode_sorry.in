{"cmd": "def f : Nat := by sorry"}

{"proofState": 0, "tactic": "sorry"}
