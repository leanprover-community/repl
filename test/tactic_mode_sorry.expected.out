{"sorries":
 [{"proofState": 0,
   "pos": {"line": 1, "column": 18},
   "goal": "⊢ Nat",
   "endPos": {"line": 1, "column": 23}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 4},
   "endPos": {"line": 1, "column": 5},
   "data": "declaration uses 'sorry'"}],
 "env": 0}

{"sorries": [{"proofState": 1, "goal": "⊢ Nat"}], "proofState": 2, "goals": []}

